frontend/src/store/admin.store.ts

```diff
@@ -0,0 +1,18 @@
+import { create } from 'zustand';
+import api from '../api';
+
+interface BearState {
```

Comment:
Oh, so you have figured it out the issue with `Zustand` in the end?! 

rename the name of the interface :D